Core/clim-core/panes.lisp

```diff
@@ -363,11 +363,38 @@ order to produce a double-click")
   (print-unreadable-object (pane sink :type t :identity t)
     (prin1 (pane-name pane) sink)))
 
+(defgeneric find-concrete-pane-class (pane-realizer pane-type)
+  (:documentation "Resolves abstract pane name to a concrete pane ~
+class which may be instatinated. Function specializes on its first ~
```

Comment:
In #929 you suggested

> Generate clause genearate-ensure-pane-form in your code should invoke make-pane-1 with the appropriate (see make-pane) realizer without trying to coerce the type on its own.

This would work best if either 
* `find-concrete-pane-class` returned a class *name*
* or `make-pane` accepted a class as the `type` argument

Reply:
A method `find-concrete-pane-class t class` that just returns the class does the job.

Reply:
this method is responsible for finding a concrete class, not a symbol of said class. if you need the class name, you may use `class-name` on the result. abstract pane names are symbols, not classes.

I've suggested, that you should not rely on ad-hoc function which slaps -pane at the end of the symbol but rather rely on the realizer resolution as specified. Only after that you have clarified, that you need the exact class before going down the make-pane road because you have a reinitialization optimization in mind.

Reply:
after a bit of thought I've came to conclusion, that allowing passing verbatim class as a pane type won't hurt. fixed.